tensorboard/plugins/interactive_inference/witwidget/pip_package/RELEASE.md

```diff
@@ -1,7 +1,10 @@
-# To release a new version of witwidget on PyPI:
+# To release a new version of witwidget on PyPI and NPM:
  
 1. Update tensorboard/plugins/interactive_inference/witwidget/version.py (set release version, remove 'dev')
-2. `bazel run tensorboard/plugins/interactive_inference/witwidget/pip_package:build_pip_package`
-3. Upload the whl files created by the previous step to PyPI as per instructions
+2. Update version number in tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/package.json
+3. `bazel run tensorboard/plugins/interactive_inference/witwidget/pip_package:build_pip_package`
+4. Upload the whl files created by the previous step to PyPI as per instructions
 at https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives.
-4. Commit the version.py change.
+5. Publish the new NPM package through running "npm publish" in the "js/" subdirectory of the package files generated in step 3.
+6. Commit the version changes.
```

Comment:
Shouldn’t the process be to commit and merge any version changes
*before* publishing to PyPI and NPM, then always publish from a
checked-in, known-good commit? Publishing immutable archives from a
dirty work tree doesn’t sound like something that we want to encourage.
Not only are you vulnerable to typos or bad local state, but others
might not be able to reliably reproduce the built artifacts. It’s much
simpler if the actual deployment process is just “clone fresh; check out
some fixed commit; run `npm publish`, et al.”.

Reply:
sounds better. updated instructions.